deployment/requirements.txt

```diff
@@ -1,61 +1,16 @@
-astroid==2.5.2
-attrs==20.3.0
-aws-requests-auth==0.4.3
-aws-sam-translator==1.45.0
-awscli==1.19.47
-blessed==1.17.6
-boto3==1.17.47
-botocore==1.20.47
-build==0.6.0.post1
-certifi==2020.12.5
-cffi==1.14.5
-cfn-lint==0.49.0
-chalice==1.26.6
-chardet==3.0.4
-click==8.1.2
-colorama==0.4.3
-crhelper==2.0.10
-cryptography==3.4.7
-decorator==4.4.2
-docker==4.2.2
-docutils==0.15.2
-idna==2.10
-inquirer==2.7.0
-isort==5.8.0
-Jinja2==2.11.3
-jmespath==0.10.0
-jsonpatch==1.32
-jsonpointer==2.2
-jsonschema==3.2.0
-junit-xml==1.9
-jwcrypto==0.8
-lazy-object-proxy==1.6.0
-MarkupSafe==1.1.1
-mccabe==0.6.1
-mypy-extensions==0.4.3
-networkx==2.5.1
-packaging==21.3
-pbr==5.8.1
-pep517==0.12.0
-pyasn1==0.4.8
-pycparser==2.20
-pyparsing==3.0.7
-pyrsistent==0.17.3
-python-dateutil==2.8.1
-python-editor==1.0.4
-pytz==2021.1
-PyYAML==5.4.1
-readchar==2.0.1
-redis==3.5.3
-requests==2.25.1
-rsa==4.7.2
-s3transfer==0.3.6
-six==1.15.0
-testresources==2.0.1
-toml==0.10.2
-tomli==2.0.1
-tomlkit==0.7.2
-urllib3==1.26.5
-wcwidth==0.2.5
-websocket-client==1.3.2
-wrapt==1.12.1
+aws-requests-auth
```

Comment:
Could we base the versions on the currently used versions and use the __compatible_ release_ clause
e.g. `awscli ~= 1.19.47`